### PR TITLE
Put the filename of the manifest file in the post-exec hook env.

### PR DESCRIPTION
**Pull request by vext01**
(also rename the function, as it is never used for pre-hooks).

OK?

### CODE REVIEW

**Comment by vext01** [comment]
Please wait on merging this.

**Comment by vext01** [comment]
Ready.

**Comment by vext01** [comment]
The failure was a network glitch in the CI. Unrelated to the change...

**Comment by snim2** [comment]
Have you restarted that build?

**Comment by vext01** [comment]
Yeah. Didn't know you could do that!

**Comment by snim2** [comment]
The only drawback is that you lose the previous log, so you don't want to use that feature for a "real" bug

File: krun/tests/test_scheduler.py [hunk 1]
```diff
@@ -237,8 +238,13 @@ def test_pre_and_post_exec_cmds0002(self, monkeypatch, mock_platform):
...
         elems = got.split(":")
-        assert elems[0].endswith(".json.bz2")
-        assert elems[1].endswith(".log")
+        assert os.path.basename(elems[0]) == "one_exec_results.json.bz2"
+        assert re.match("one_exec_[0-9]{8}_[0-9]{6}.log", os.path.basename(elems[1]))
+        assert os.path.basename(elems[2]) == "one_exec.manifest"
+
+        # all paths should be in the same dir
+        dirnames = [os.path.dirname(x) for x in elems]
+        assert all([dirnames[i] == dirnames[0] for i in [1, 2]])
```
Comment (ltratt) [inline krun/tests/test_scheduler.py]: This `assert` feels weird, particularly with the only-for-the-assert variable above it. Could we not do:

```python
assert os.path.dirname(dirnames[0]) == os.path.dirname(dirnames[1])
assert os.path.dirname(dirnames[0]) == os.path.dirname(dirnames[2])
```

**Comment by vext01** [comment]
Try this. OK to squash?

**Comment by ltratt** [comment]
Ah, that's a bit neater. Now if we do:

```python
assert os.path.dirname(elems[0]) == os.path.dirname(elems[1]) == os.path.dirname(elems[2])
```

I think we have a winner.

**Comment by vext01** [comment]
Here we go. (I find this less readable)

**Comment by ltratt** [comment]
I prefer this because now there's not a variable floating around that's only used by an `assert`. But I don't feel *that* strongly about it. Squash as you feel fit, and feel free to use whichever version of the `assert` you prefer.

**Comment by vext01** [comment]
Squashed.

**Comment by ltratt** [comment]
Will merge when tests have run.

**Comment by vext01** [comment]
Once this is merged, there is a partner commit here: https://github.com/softdevteam/warmup_experiment/pull/209

**Pull request merged**